liberty-ls/src/main/resources/BootstrapProperties.properties

```diff
@@ -0,0 +1,38 @@
+com.ibm.ws.logging.isoDateFormat=This setting specifies whether to use ISO-8601 formatted dates in log files. The default value is `false`. If this setting is set to `true`, the ISO-8601 format is used in the messages.log file, the trace.log file, and the FFDC logs. The format is `yyyy-MM-dd'T'HH:mm:ss.SSSZ`. If you specify a value of `false`, the date and time are formatted according to the default locale set in the system. If the default locale is not found, the format is `dd/MMM/yyyy HH:mm:ss:SSS z`
+com.ibm.ws.logging.max.files=This setting specifies how many of each of the logs files are kept. This setting also applies to the number of exception summary logs for FFDC. So if this number is 10, you might have 10 message logs, 10 trace logs, and 10 exception summaries in the ffdc directory. By default, the value is `2`. The console log does not roll so this setting does not apply to the console.log file.
+com.ibm.ws.logging.max.file.size=This setting specifies the maximum size (in MB) that a log file can reach before it is rolled. Setting the value to `0` disables log rolling. The default value is `20`. The console.log does not roll so this setting does not apply.
+com.ibm.ws.logging.message.file.name=This setting specifies the name of the message log file. The message log file has a default name of `messages.log`. This file always exists, and contains INFO and other (AUDIT, WARNING, ERROR, FAILURE) messages in addition to the System.out and System.err streams. This log also contains time stamps and the issuing thread ID. If the log file is rolled over, the names of earlier log files have the format messages_timestamp.log
+com.ibm.ws.logging.message.format=WLP_LOGGING_MESSAGE_FORMAT
+com.ibm.ws.logging.message.source=WLP_LOGGING_MESSAGE_SOURCE
+com.ibm.ws.logging.trace.file.name=This setting specifies the name of the trace log file. The trace.log file is created only if additional or detailed trace is enabled. `stdout` is recognized as a special value, and causes trace to be directed to the original standard out stream.
+com.ibm.ws.logging.trace.format=This setting controls the format of the trace log. The default format for Liberty is `ENHANCED`. You can also use `BASIC` and `ADVANCED` formats
+com.ibm.ws.logging.trace.specification=This setting is used to selectively enable trace. The log detail level specification is in the following format: `component = level`
+com.ibm.ws.logging.apps.write.json=WLP_LOGGING_APPS_WRITE_JSON
```

Comment:
I think this is going to cause issues with the translation folks. They won't understand how to translate this. We may just need to keep a Map in Messages.java that has this key/value pairing. If the key contains a `.` and is in that map, then get the value from the map and use it to retrieve the message from the other properties file.

Reply:
I've gone ahead and implemented this locally, but I don't like how the bootstrap properties are split up between the `BootstrapProperties.properties` file and a separate Map that contains the equivalent property pairings.

Reply:
Just recording here what we discussed in Slack. It will be less confusing for the translators if we move the special handling for these properties into Messages.java. That way we will not have messages in the properties file that should not be translated. These properties already had special behavior in Messages.java anyway.